jellyfish/api/_room_api.py

```diff
@@ -135,7 +137,14 @@ def add_component(
             ComponentOptionsSIP,
         ],
     ) -> Union[ComponentFile, ComponentHLS, ComponentRTSP, ComponentSIP]:
-        """Creates component in the room"""
+        """
+        Creates component in the room.
+        Currently there are 3 different components:
+        * File Component which options are `ComponentOptionsFile`
```

Comment:
```suggestion
        * File Component for which the options are `ComponentOptionsFile`
```
and the rest likewise